.travis.yml

```diff
@@ -34,14 +34,10 @@ install:
   - nvm use $TARGET_NODE_VERSION
   - node --version
   - npm --version
-  - npm install -g elm@0.18.0 elm-test@0.18.7 elm-verify-examples@1.0.2 elm-format@exp
-  - mv $(npm config get prefix)/bin/elm-make $(npm config get prefix)/bin/elm-make-old
-  - printf '%s\n\n' '#!/bin/bash' 'echo "Running elm-make with sysconfcpus -n 2"' '$TRAVIS_BUILD_DIR/sysconfcpus/bin/sysconfcpus -n 2 elm-make-old "$@"' > $(npm config get prefix)/bin/elm-make
-  - chmod +x $(npm config get prefix)/bin/elm-make
-  - travis_retry elm-package install --yes
-  - cd tests
-  - travis_retry elm-package install --yes
-  - cd ..
+  - npm install -g elm@0.19.0 elm-test@0.19.0-rev4 elm-verify-examples@3.0.1 elm-format@elm0.19.0
```

Comment:
Note this integration -- https://blog.travis-ci.com/2018-11-20-elm-is-now-supported-on-travis-ci

Reply:
@amitaibu Thank you. This greatly simplified the ci config!